backend/src/api/UserController.ts

```diff
@@ -65,6 +67,10 @@ export default class UserController {
             bio: Joi.string().required().max(1024 * 4)
         });
 
+        const nameSchema = Joi.object<UserSaveNameRequest>({
+            name: Joi.string().required()
```

Comment:
Probably a good idea to add a length check here (and also in https://github.com/spaceshelter/orbitar/blob/df2e5f892c9bf810086a117908677b2b96a66d70/backend/src/api/InviteController.ts#L50).

`username` is currently a `varchar(50)`

Reply:
done